google-cloud-errorreporting/src/test/java/com/google/cloud/errorreporting/spi/v1beta1/MockErrorGroupServiceImpl.java

```diff
@@ -41,8 +40,16 @@ public MockErrorGroupServiceImpl() {
     return requests;
   }
 
+  public void addResponse(GeneratedMessageV3 response) {
```

Comment:
Could you add `@Override` to `addResponse` and `addException` to be consistent with `Mock*GroupService`?